packages/assets/src/scss/styles.scss

```diff
@@ -15,6 +15,7 @@
 @use 'input';
 @use 'inputs-list';
 @use 'label';
+@use 'badge';
```

Comment:
ORDER!
Ok, now to be serious. ;) I'd prefer if uses would be in alphabetical order, will be easier for later work.